src/com/cdd/bao/template/ModelSchema.java

```diff
@@ -123,7 +124,7 @@ datastructure in a local file (using Turtle format), or to push & pull between a
 
 	// ------------ static methods ------------	
 	
-	private static final String[] PREFIX_MAP = new String[]
+	public static String[] prefix_map = new String[]
```

Comment:
Use proper variable naming convention.

Reply:
We are coerced by styling requirements (ie, `checkstyle`) to use lower-case on non-`final` variable names.  The reason we made `prefix_map` non-final is because we now have to
supplement the default list with the (prefix, uri) pair from the `provisional` configuration in `config.json`.

Reply:
Yeah - use camel case, i.e. `prefixMap`.